src/catalog/postgres/pg_proc_impl.cpp

```diff
@@ -550,7 +550,7 @@ void PgProcImpl::BootstrapProcContexts(const common::ManagedPointer<transaction:
   create_fn("upper", VAR, {VAR}, execution::ast::Builtin::Upper, true);
 
   // Replication.
-  create_fn("replication_get_last_record_id", INT, {}, execution::ast::Builtin::ReplicationGetLastRecordId, true);
+  create_fn("replication_get_last_txn_id", INT, {}, execution::ast::Builtin::ReplicationGetLastTransactionId, true);
```

Comment:
Having to add two separate definitions is annoying, but I am overall very pleased with how adding something to pg_proc works now.